### ISSUE TITLE
500 Error on Webpage

### ISSUE THREAD

**Issue by runmaxde** [saadeghi/daisyui#2417]
### 1. Checklist

- [X] I updated daisyUI (`npm i -D daisyui@latest`) but I still have this issue
- [X] I looked up into [FAQ](https://github.com/saadeghi/daisyui/discussions/1949) but I couldn't find a solution
- [X] I searched in [existing issues](https://github.com/saadeghi/daisyui/issues?q=is%3Aissue) and [discussion forum](https://github.com/saadeghi/daisyui/discussions?discussions_q=) but this bug has not addressed before
- [X] This bug is dasiyUI issue and it's **NOT** specific to a JS framework or a build tool."

### 3. Which browsers are you seeing the problem on?

Chrome

### 4. Reproduction URL

https://daisyui.com/components/menu/

### 5. Describe your issue

The webpage shows 500 Error on nearly all Pages

**Comment by saadeghi** [comment]
Fixed

**Comment by Benedict-Lojewski** [comment]
Same issue here. Documentation is down 500 Error :(

**Comment by NicoBorsdorf** [comment]
https://daisyui.com/theme-generator/

Sadly not working for me, is it a DNS issue?

**Comment by saadeghi** [comment]
> is it a DNS issue?

Yes. I think accidentally you opened the page in the exact second the site was being deployed 😅  
It's rare but it happens.

It would be okay if you do a refresh without cache or if you clear site data or if you open on a private tab.
